src/content/1.7/development/components/grid/columns-reference/link.md

```diff
@@ -0,0 +1,38 @@
+---
+title: LinkColumn reference
+menuTitle: LinkColumn
+weight: 10
+---
+
+# LinkColumn Type
+{{< minver v="1.7.6" title="true" >}}
+
+This LinkColumn displays a raw field data encapsulated in a link (very useful to add an edition link on a name for example).
+
+## Available options
+
+| Properties     | Type   | Expected value                                                                     |
+| -------------- | ------ | ---------------------------------------------------------------------------------- |
+| **field**      | string | **required** Record field name which column displays.                              |
+| **route** | string | **required** Route used to generated link url. |
+| **route_param_name** | string | **required** Parameter name used by the route to generate the url. |
+| **route_param_field** | string | **required** Record field containing the route parameter. |
+
+## Example usage
+
+```php
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\LinkColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+
+$linkColumn = new LinkColumn('total_spent');
```

Comment:
Change the identifier and the name to be consistent with others?

Reply:
yep my bad it was copied from badge